src/components/common/BookCombo.js

```diff
@@ -30,7 +30,7 @@ const useStyles = makeStyles((theme) => ({
     textTransform: "capitalize",
     backgroundColor: "#fff",
     border: "1px solid #fff",
-    [theme.breakpoints.only("xs")]: {
+    [theme.breakpoints.down("sm")]: {
```

Comment:
This can be kept as xs as in the landing page we are not changing things
```suggestion
    [theme.breakpoints.down("xs")]: {
```